ios/RCTBridge+UIScene.m

```diff
@@ -12,6 +12,14 @@ @implementation RCTBridge (UIScene)
 - (instancetype)initWithDelegate:(id<RCTBridgeDelegate>)delegate
                connectionOptions:(UISceneConnectionOptions *)connectionOptions
 {
+    if (connectionOptions.notificationResponse) {
+        NSDictionary<UIApplicationLaunchOptionsKey, id> *launchOptions = @{
+             UIApplicationLaunchOptionsRemoteNotificationKey: connectionOptions.notificationResponse.notification.request.content.userInfo,
+        };
+
+        return [self initWithDelegate:delegate launchOptions:launchOptions];
```

Comment:
Why are we returning early here? Shouldn't we be sending this information along with any other information in `connectionOptions`?

Or would that scenario never happen?